lib/SPIRV/libSPIRV/SPIRV.debug.h

```diff
@@ -991,7 +991,10 @@ inline bool hasDbgInstParentScopeIdx(
     ParentScopeIdx = TypeMember::OpenCL::ParentIdx;
     return true;
   case SPIRVDebug::TypeInheritance:
-    ParentScopeIdx = TypeInheritance::ParentIdx;
+    if (ExtKind == SPIRV::SPIRVEIS_OpenCL_DebugInfo_100)
+      ParentScopeIdx = TypeInheritance::ParentIdx;
+    else
+      ParentScopeIdx = TypeInheritance::ChildIdx;
```

Comment:
It'd be better to use something like `TypeInheritance::NonSemantic::ParentIdx`. Do you mind to rework this?

Reply:
Done.